src/routes/+layout.svelte

```diff
@@ -3,8 +3,12 @@
     import NavList from './NavList.svelte';
     import SidesFooter from '$lib/components/SidesFooter.svelte';
     import upcsi from '$lib/icons/upcsi.svg';
+    import Navbar from '$lib/components/NavbarComponent.svelte';
```

Comment:
This import should be sorted above `NavList`. Also do consider renaming it to `NavBar` for consistency. Maybe we can even rename the file as well.
```suggestion
    import NavBar from '$lib/components/NavBar.svelte';
```